data/mods/ssb/abilities.js

```diff
@@ -383,6 +383,25 @@ let BattleAbilities = {
 			pokemon.illusion = null;
 		},
 	},
+	// Gallant Spear
+	trombe: {
+		desc: "On switch in, this Pokemon summons Light Screen or Reflect and is guaranteed to move first.",
+		shortDesc: "On switch, 1 screen + priority.",
+		id: "trombe",
+		name: "Trombe!",
+		isNonstandard: "Custom",
+		onSwitchIn(pokemon) {
+			this.add('-activate', pokemon, 'ability: Trombe!');
+			if (Math.floor(Math.random() * 2) === 0) {
```

Comment:
```suggestion
			if (Math.round(this.random())) {
```
Again, no Math.random().